src/dropzone/DropzoneDirective.ts

```diff
@@ -92,5 +105,24 @@ export class DropzoneDirective implements ng.IDirective {
     angular.forEach(eventHandlers, (handler, event) => {
       dropzone.on(event, handler);
     });
+
+    Dropzone.prototype.uploadFiles = function(files: any) {
```

Comment:
Add comments to elaborate on the override.

Reply:
Thanks for pointing this out, a comment has been added to this override.